src/Microsoft.ML.PCA/PcaTrainer.cs

```diff
@@ -41,7 +41,7 @@ namespace Microsoft.ML.Trainers.PCA
     /// <remarks>
     /// This PCA can be made into Kernel PCA by using Random Fourier Features transform
     /// </remarks>
-    public sealed class RandomizedPcaTrainer : TrainerEstimatorBase<AnomalyPredictionTransformer<PcaPredictor>, PcaPredictor>
+    public sealed class RandomizedPcaTrainer : TrainerEstimatorBase<AnomalyPredictionTransformer<PcaModelParameters>, PcaModelParameters>
     {
         public const string LoadNameValue = "pcaAnomaly";
```

Comment:
>public [](start = 8, length = 6)

can it become internal?

Reply:
+1

---
In reply to: [242419333](https://github.com/dotnet/machinelearning/pull/1901#discussion_r242419333) [](ancestors = 242419333)